package.json

```diff
@@ -72,7 +72,7 @@
       },
       {
         "path": "./lib/all.js",
-        "maxSize": "280kb"
+        "maxSize": "290kb"
```

Comment:
has the bundle size eventually increased by 10 kb? I am assuming this is to the all bundle size?

Reply:
yes, only for `/all` because it contains chunks that now contain some "inline" styles. the core package didn't change